MediaGalleryIntegration/Plugin/SaveBaseCategoryImageInformation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryIntegration\Plugin;
+
+use Magento\MediaGalleryApi\Api\SaveAssetsInterface;
+use Magento\Catalog\Model\ImageUploader;
+use Magento\MediaGallerySynchronization\Model\CreateAssetFromFile;
+use Magento\Cms\Model\Wysiwyg\Images\Storage;
+use Magento\MediaGalleryUi\Model\Filesystem\SplFileInfoFactory;
```

Comment:
Can you please move `SplFileInfoFactory` to `MediaGallerySynchronization` module and remove the dependency on `MediaGalleryUi` module

Reply:
ok